conditional-signing/nodejs/src/index.js

```diff
@@ -69,7 +69,7 @@ function getWallet(privateKey) {
     return new Wallet(
       privateKey,
       new ethersProviders.JsonRpcProvider(
-        "https://chain-rpc.litprotocol.com/http"
+        'https://vesuvius-rpc.litprotocol.com'
```

Comment:
Can we pull this from the [constants package](https://github.com/LIT-Protocol/js-sdk/blob/f555777b4d4e0e6333b87ac83df7b1b1c7c68e01/packages/constants/src/lib/constants/constants.ts#L633) instead of specifying the URL